package.json

```diff
@@ -46,10 +46,16 @@
+    "compat:mobile": "npm-run-all -c -p --max-parallel 5 compat:ios:iphone_xs compat:ios:iphone8 compat:android:pixel3 compat:android:pixel compat:android:galaxys8",
+    "compat:ios:iphone_xs": "cross-env BROWSER_STACK_DEVICE='iPhone XS' BROWSER_STACK_OS_VERSION=12.1 BROWSER_STACK_BROWSER=iPhone cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config/environment.js' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/ios-iPhone-xs-report.json",
+    "compat:ios:iphone8": "cross-env BROWSER_STACK_DEVICE='iPhone 8' BROWSER_STACK_OS_VERSION=11.0 BROWSER_STACK_BROWSER=iPhone cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config/environment.js' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/ios-iPhone-8-report.json",
     "compat:android:pixel3": "cross-env BROWSER_STACK_DEVICE='Google Pixel 3' BROWSER_STACK_OS_VERSION=9.0 BROWSER_STACK_BROWSER=android cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config/environment.js' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/android-pixel3-report.json",
-    "compat:android:galaxys8": "cross-env BROWSER_STACK_DEVICE='Samsung Galaxy S8' BROWSER_STACK_OS_VERSION=7.0 BROWSER_STACK_BROWSER=android cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config/environment.js' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/android-galaxys8-report.json"
+    "compat:android:pixel": "cross-env BROWSER_STACK_DEVICE='Google Pixel' BROWSER_STACK_OS_VERSION=8.0 BROWSER_STACK_BROWSER=android cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config/environment.js' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/android-pixel-report.json",
+    "compat:android:galaxys8": "cross-env BROWSER_STACK_DEVICE='Samsung Galaxy S8' BROWSER_STACK_OS_VERSION=7.0 BROWSER_STACK_BROWSER=android cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config/environment.js' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility/android-galaxys8-report.json",
+    "compat:tablet": "npm-run-all -c -p --max-parallel 5 compat:ios:ipad6 compat:ios:ipad_pro_11 compat:ios:ipad_mini_4",
```

Comment:
nexus 7 is not included in browserstack automate, so added the next three most popular tablets.